integrations/azure/azure_integration/overrides.py

```diff
@@ -0,0 +1,17 @@
+from port_ocean.core.handlers.port_app_config.models import (
+    BaseModel,
+    ResourceConfig,
+    PortAppConfig,
+)
+
+
+class AzureResourceConfig(ResourceConfig):
+    class Selector(BaseModel):
+        query: str
+        api_version: str
```

Comment:
Is it required?
Is there a way to get the latest version if the api version was not specified?

Reply:
yes, each resource has its own supported api versions, and there is no support for latest. 
As a side note we would like to avoid using latest so we won't receive a response with different structure than we are expecting.